head/src/main/kotlin/io/qalipsis/core/head/jdbc/repository/ReportTaskRepository.kt

```diff
@@ -34,11 +35,18 @@ import io.qalipsis.core.head.jdbc.entity.ReportTaskEntity
 @R2dbcRepository(dialect = Dialect.POSTGRES)
 @Requires(notEnv = [ExecutionEnvironments.TRANSIENT])
 internal interface ReportTaskRepository : CoroutineCrudRepository<ReportTaskEntity, Long> {
+
     /**
      * Returns the [ReportTaskEntity] that matches the id and creator.
      */
     suspend fun findByTenantReferenceAndReference(
         tenant: String,
         reference: String
     ): ReportTaskEntity?
+
+
+    /**
+     * Deletes all records of report task less than the given minimalTaskExpiryDate.
+     */
+    suspend fun deleteAllByCreationTimestampLessThan(minimalTaskExpiryDate: Instant)
```

Comment:
While it is ok to use the creation timestamp for the files, I would rather use the version timestamp for the tasks, because there might be a delay between their creation and completion.

Reply:
By their version Timestamp do you mean update timestamp

Reply:
Indeed.